src/index.html

```diff
@@ -13,6 +13,46 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image-img"
+        src="../src/images/imac.jpeg"
+        alt="mac"
+      />
+
+      <p class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__stars">
+        <div class="card__stars-star card__stars-star--4">
+          <div class="card__stars-star-item"></div>
```

Comment:
Fix classes everywhere
```suggestion
          <div class="stars__star"></div>
```